### PR TITLE
Regex update to UAT

### PR DESCRIPTION
**Pull request by andersmx**

### CODE REVIEW

File: json/stage-json/foodPantry.json [hunk 1]
```diff
@@ -4676,7 +4676,7 @@
           "tableView": false,
           "clearOnHide": false,
           "validate": {
-            "custom": "(function () {\n  // check if not a valid character\n  if (/[|;$%\"<>{}+/]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these | ; $ % \" < >{ } + /';\n    return;\n  }\n\n  valid = true;\n})();"
+            "custom": "(function () {\n  // check if not a valid character\n  if (/[|&;$%\"<>(){}+]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these | ; $ % \" < >{ } +';\n    return;\n  }\n\n  valid = true;\n})();"
```
Comment (RossWebsterWork) [inline json/stage-json/foodPantry.json:4679]: Brackets and ampersand have been added to the regex but not to the validation message

File: json/stage-json/foodPantry.json [hunk 2]
```diff
@@ -4761,7 +4761,7 @@
           "tableView": false,
           "clearOnHide": false,
           "validate": {
-            "custom": "(function () {\n  // check if not a valid character\n  if (/[|;$%\"<>{}+/]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these | ; $ % \" < >{ } + /';\n    return;\n  }\n\n  valid = true;\n})();"
+            "custom": "(function () {\n  // check if not a valid character\n  if (/[&;$%\"<>(){}+]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these & ; $ % \" < >{ } +';\n    return;\n  }\n\n  valid = true;\n})();"
```
Comment (RossWebsterWork) [inline json/stage-json/foodPantry.json:4764]: Brackets have been added to the regex but not to the validation message

**Review by RossWebsterWork — APPROVED** [review]

File: json/stage-json/foodPantry.json (hunk 2 above)
Comment (andersmx) [reply json/stage-json/foodPantry.json]: Now resolved

File: json/stage-json/foodPantry.json (hunk 1 above)
Comment (andersmx) [reply json/stage-json/foodPantry.json]: Now resolved

**Review by ganapavz — APPROVED** [review]

**Pull request merged**